dbt_subprojects/tokens/models/prices/bnb/prices_bnb_tokens.sql

```diff
@@ -290,7 +290,8 @@ FROM
     ('ice-ice-network', 'bnb', 'ICE', 0xc335df7c25b72eec661d5aa32a7c2b7b2a1d1874, 18),
     ('jmpt-jumptoken', 'bnb', 'JMPT', 0x88d7e9b65dc24cf54f5edef929225fc3e1580c25, 18),
     ('usdfi-usdfi', 'bnb', 'USDFI', 0x7df1938170869afe410098540c051a8a50308988, 18),
-    ('wncg-wrapped-ncg', 'bnb', 'WNCG', 0x52242cbAb41e290E9E17CCC50Cc437bB60020a9d, 18)
+    ('wncg-wrapped-ncg', 'bnb', 'WNCG', 0x52242cbAb41e290E9E17CCC50Cc437bB60020a9d, 18),
+    ('atlas-star-atlas-wormhole', 'bnb', 'ATLAS', 0x83850d97018f665eb746fbb8f18351e977d1b0d6, 8)    
```

Comment:
looks like the API token_id value didn't match URL value. i copied from the API key section in that linked coinpaprika page and pasted here. this should work